src/DummyFileBasedCms.Web/MvcInfrastructureStuff/FlatFileCmsRazorProjectPageRouteModelProvider.cs

```diff
@@ -1,145 +0,0 @@
-﻿namespace DummyFileBasedCms.Web.MvcInfrastructureStuff
-{
-    using System;
-    using System.IO;
-    using System.Linq;
-    using Microsoft.AspNetCore.Hosting;
-    using Microsoft.AspNetCore.Mvc.ApplicationModels;
-    using Microsoft.AspNetCore.Mvc.RazorPages;
-    using Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure;
-    using Microsoft.AspNetCore.Mvc.RazorPages.Internal;
-    using Microsoft.AspNetCore.Razor.Language;
-    using Microsoft.Extensions.Logging;
-    using Microsoft.Extensions.Options;
-
-    public class FlatFileCmsRazorProjectPageRouteModelProvider : IPageRouteModelProvider
```

Comment:
Implementing an `IPageRouteModelProvider` might still be useful if you want to customize how pages in the external directory behave. In `OnProvidersExecuted` you should be able to see and modify what was created by the default providers.